package.json

```diff
@@ -188,7 +188,10 @@
     "ts-node": "^10.9.2",
     "typescript": "~5.3.3"
   },
+  "overrides": {
+    "better-sqlite3": "$better-sqlite3"
```

Comment:
This is just a weird way to write "use the version for `better-sqlite3` specified above in `dependencies`", so it doesn't have to be kept in sync if the version range gets upgraded again in the future.